resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -439,18 +439,31 @@ export default {
             Events.$emit('images.change', this.imageId, this.image);
         },
         cachePreviousAndNext() {
+            // currently const number of images to cache
+            let imagesToLoad = 10;
+            if (imagesToLoad > this.allImagesIds.length) {
```

Comment:
I checked and you don't actually need to handle this special case. `getNextIndex` just wraps around if the index gets too large. This is desired behavior, too, as these images are also next in the "queue" and should be cached.